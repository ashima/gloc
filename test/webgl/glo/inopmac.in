#if EXTMAC
#define MYMAC vec4(1.)
#else
#define MYMAC vec4(0.)
#endif

void main() { gl_Position = MYMAC; }
