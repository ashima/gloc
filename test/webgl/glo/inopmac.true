{
  "glo": [ 0, 8, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "author": {},
    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [],
      "outsym": [ "main" ],
      "inmac": [ "EXTMAC" ],
      "opmac": [ "MYMAC" ],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\n#if EXTMAC\n#define MYMAC vec4(1.)\n#else\n#define MYMAC vec4(0.)\n#endif\n\nvoid main() { gl_Position = MYMAC; }"
    }
  ],
  "linkmap": {}
}
