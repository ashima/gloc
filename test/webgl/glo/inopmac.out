{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "copyright": [ "", 2011, "" ],
    "author": {},
    "license": null,
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [],
      "outsym": [ "main" ],
      "inmac": [ "EXTMAC" ],
      "opmac": [ "MYMAC" ],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\n#if EXTMAC\n#define MYMAC vec4(1.)\n#else\n#define MYMAC vec4(0.)\n \n#endif\n\nvoid main() { gl_Position = MYMAC; }"
    }
  ],
  "linkmap": { "0": "<stdin>#0" }
}
